dask/dataframe/core.py

```diff
@@ -577,6 +577,40 @@ def _aca_agg(self, token, func, aggfunc=None):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def rolling(self, window, min_periods=None, win_type=None):
+        """Provides rolling transformations.
+
+        Parameters
+        ----------
+
+        window: int
+            Size of the moving window. This window must be smaller
+            than the size of the previous partition.
+        min_periods: int, default None
+            Minimum number of observations in window required to have
+            a value (otherwise result is NA).
+        win_type: string, default None
+            Provide a window type. (Identical to pandas.)
+
+        The center, freq, and axis arguments are not supported.
+        """
+        from dask.dataframe.rolling import Rolling
+
+        if not isinstance(window, int):
```

Comment:
Right. The problem is that we only construct the `.rolling` object during computation.  This error might happen quite a bit later and quite possibly on another machine.  It's nice to raise the error when the user enters the code rather than possibly quite a bit later.